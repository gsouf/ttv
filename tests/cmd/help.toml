bin.name = "ttv"
args = "-h"
