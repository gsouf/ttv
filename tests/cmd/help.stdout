ttv 0.4.0
Flexibly create test, train and validation sets

USAGE:
    ttv [FLAGS] <SUBCOMMAND>

FLAGS:
    -h, --help       Prints help information
    -V, --version    Prints version information
    -v               Set the level of verbosity

SUBCOMMANDS:
    help     Prints this message or the help of the given subcommand(s)
    split    Split dataset into two or more files for test/train/validation sets
